internal/types/types.go

```diff
@@ -84,6 +84,15 @@ type Repo struct {
 	KeyValuePairs map[string]*string `json:",omitempty"`
 }
 
+const (
+	ReposDomain   GitHubAppDomain = "repos"
+	BatchesDomain GitHubAppDomain = "batches"
+)
+
+type GitHubAppDomain string
+
+func (s GitHubAppDomain) ToGraphQL() string { return strings.ToLower(string(s)) }
```

Comment:
Oops, did I give you this? I think it's backwards ahaha:

```suggestion
func (s GitHubAppDomain) ToGraphQL() string { return strings.ToUpper(string(s)) }
```

I think that's why the buildkite test is failing too.